Makefile

```diff
@@ -153,6 +150,10 @@ KUSTOMIZE = $(shell pwd)/bin/kustomize
 kustomize: ## Download kustomize locally if necessary.
 	$(call go-get-tool,$(KUSTOMIZE),sigs.k8s.io/kustomize/kustomize/v4@v4.5.5)
 
+ENVTEST = $(shell pwd)/bin/setup-envtest
```

Comment:
Also, seems to need quotes here?

Reply:
we don't have that in any of the other installations do you want me to add?

Reply:
should be ok as it is now, but just something to keep in mind